File: crates/arborium-fsharp/sample.fs
module App

type Person = { Name: string; Age: int }

let greet person =
    sprintf "Hello, %s!" person.Name

let people = [
    { Name = "Alice"; Age = 30 }
    { Name = "Bob"; Age = 25 }
]

people
|> List.map greet
|> List.iter (printfn "%s")

File: langs/group-cedar/scala/def/sample.scala
object Main extends App {
  case class Person(name: String, age: Int)

  val people = List(
    Person("Alice", 30),
    Person("Bob", 25)
  )

  people
    .filter(_.age > 20)
    .map(p => s"Hello, ${p.name}!")
    .foreach(println)
}
